modes/segmentation/src/index.tsx

```diff
@@ -156,7 +156,7 @@ function modeFactory({ modeConfiguration }) {
     /** List of extensions that are used by the mode */
     extensions: extensionDependencies,
     /** HangingProtocol used by the mode */
-    // hangingProtocol: [''],
+    // hangingProtocol: ['@ohif/mnGrid'],
```

Comment:
I guess you wanted to include mnGrid right?

Reply:
It doesn't really work to make it an empty string, so I wanted an example that is what we normally get by default.

Reply:
Added a comment